modules/flann/include/opencv2/flann/lsh_index.h

```diff
@@ -378,11 +378,11 @@ class LshIndex : public NNIndex<Distance>
     IndexParams index_params_;
 
     /** table number */
-    unsigned int table_number_;
+    int table_number_;
     /** key size */
-    unsigned int key_size_;
+    int key_size_;
     /** How far should we look for neighbors in multi-probe LSH */
-    unsigned int multi_probe_level_;
+    int multi_probe_level_;
```

Comment:
Please revert these changes too

Reply:
I've squashed all these commits in a new branch pev--fix-bad-any-cast-in-lsh based on 3.4, including a test.
So we don't change any unsigned int for int type for table_number_, key_size_ and multi_probe_level_.
However, as
- in all the others types of IndexParams the types are int and not unsigned int
- and the version of these three parameters in LshIndexParams of miniflann is int and not unsigned int,
I would suggest for consistency to switch to int in master branch, what is actually done by the "pev-fix-lsh-bad-any-cast" (one dash and not two after pev to easily identify between master based branched and 3.4 based ones)